admin/tests-integration/sql.service.spec.js

```diff
@@ -178,6 +178,17 @@ describe('sql.service:integration', () => {
     expect(typeof row).toBe('object')
   })
 
+  xit('#findOneByPupilId should retrieve a row', async () => {
```

Comment:
please remove. these should not be in the sql service

Reply:
@GuyHarwood removed